.github/main.workflow

```diff
@@ -0,0 +1,11 @@
+workflow "Documentation" {
+  on = "push"
+  resolves = ["Build and Deploy Jekyll"]
+}
+
+action "Build and Deploy Jekyll" {
```

Comment:
still learning github actions should this be `action “git.master”`

Reply:
I am also still learning github actions, as far as I can tell this string can be anything as long as it matches what you call in "resolves"